### PR TITLE
Added dummy data for model testing and changed settings

### PR DESCRIPTION
**Pull request by gaurKrishna**
Dummy data created for content models and tested api for Course model.

### CODE REVIEW

File: api/settings.py [hunk 1]
```diff
@@ -23,8 +23,10 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config('SECRET_KEY')
 
+
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=True, cast=bool)
+#DEBUG = config('DEBUG', default=True, cast=bool)
```
Comment (theyashshahs) [inline api/settings.py]: can you explain this change

Comment (gaurKrishna) [reply api/settings.py]: Config file was not working on my laptop so I changed those settings, I will take care of removing my settings and restore to previous one in my next pull request.

Comment (theyashshahs) [reply api/settings.py]: read about `python-decouple` and how it is used in the django projects, you will not need to manage env var

Comment (gaurKrishna) [reply api/settings.py]: Ok I will work on it.

**Pull request closed without merging**